src/patternfly/components/Switch/switch.scss

```diff
@@ -4,13 +4,16 @@
   --pf-c-switch--Height: calc(#{var(--pf-global--FontSize--md)} * #{var(--pf-global--LineHeight--md)});
   --pf-c-switch--Width: calc(#{var(--pf-c-switch--Height)} * #{var(--pf-global--golden-ratio)});
   --pf-c-switch--LineHeight: var(--pf-global--LineHeight--md);
-  --pf-c-switch__input--checked__toggle--BackgroundColor: var(--pf-global--primary-color--100);
-  --pf-c-switch__input--checked__toggle--before--Transform: translateX(calc(#{var(--pf-c-switch--Width)} - #{var(--pf-c-switch__toggle--before--Width)} - 2 * #{var(--pf-c-switch__toggle--Padding)}));
-  --pf-c-switch__input--checked__label--Color: var(--pf-global--Color--dark-100);
-  --pf-c-switch__input--not-checked__label--Color: var(--pf-global--disabled-color--100);
-  --pf-c-switch__input--disabled__label--Color: var(--pf-global--disabled-color--100);
-  --pf-c-switch__input--disabled__toggle--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-200);
-  --pf-c-switch__input--disabled__toggle--before--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+
+  // Switch Input Checked Toggle
+  --pf-c-switch__toggle--checked--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-switch__toggle--checked--before--Transform: translateX(calc(#{var(--pf-c-switch--Width)} - #{var(--pf-c-switch__toggle--before--Width)} - 2 * #{var(--pf-c-switch__toggle--Padding)}));
+
+  // Switch Input
+  --pf-c-switch__toggle--disabled--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-200);
```

Comment:
Actually just noticed here, the original var was correct. The var name should follow the selectors BEM-style, so 

`--pf-c-switch__input--disabled__label--Color` means

```
.pf-c-switch__input:disabled
  .pf-c-switch__label
    color:
```

Honestly, I _think_ the way the stylesheet was before was correct :) Do you see any errors in the var names? I do like the changes you made to the `~ .pf-c-switch__toggle` blocks by using a single selector instead of 2.